backpack/core/derivatives/lstm.py

```diff
@@ -47,8 +47,8 @@ def _check_parameters(module: LSTM) -> None:
             raise NotImplementedError("only dropout = 0 is supported")
         if module.bidirectional is not False:
             raise NotImplementedError("only bidirectional = False is supported")
-        if module.proj_size != 0:
-            raise NotImplementedError("only proj_size = 0 is supported")
+        # if module.proj_size != 0:
+        #     raise NotImplementedError("only proj_size = 0 is supported")
```

Comment:
Revert. I commented it out to circumvent https://github.com/fKunstner/backpack-discuss/issues/112, but forgot to reactivate the code.